source/includes/data-formats/bson.cpp

```diff
@@ -0,0 +1,88 @@
+#include <iostream>
+#include <chrono>
+
+#include <bsoncxx/builder/basic/document.hpp>
+#include <bsoncxx/builder/list.hpp>
+#include <bsoncxx/builder/stream/document.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/client.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/uri.hpp>
+
+using bsoncxx::builder::basic::kvp;
+using bsoncxx::builder::basic::make_document;
+
+int main() {
+    mongocxx::instance instance;
+    mongocxx::uri uri("<connection string>");
+    mongocxx::client client(uri);
+
+    {
+        // Creates a BSON document using the list builder
+        // start-bson-list
+        bsoncxx::builder::list course_doc = { "title", "Poetry",
+                                              "department", "English" };
+
+        bsoncxx::builder::list courses_array = { "Poetry", "Literature",
+                                                 "Creative Writing" };
+
+        bsoncxx::document::value course{course_doc.view().get_document()};
```

Comment:
Oop. The suggestion in https://github.com/mongodb/docs-cpp/pull/81#discussion_r1878793949 was missing a `.value` to access the `bsoncxx::document::view`:

```suggestion
        bsoncxx::document::value course{course_doc.view().get_document().value};
```

To test compile locally, one quick way is to use `pkg-config` to get the compiler flags:
```
$ brew install mongo-cxx-driver
$ clang++ -std=c++17 $(pkg-config --cflags --libs libmongocxx) -o main main.cpp
```

Reply:
ah thank you!